src/uct/sm/mm/mm_iface.c

```diff
@@ -37,6 +37,10 @@ static ucs_config_field_t uct_mm_iface_config_table[] = {
      " try - Try to allocate memory using huge pages and if it fails, allocate regular pages.\n",
      ucs_offsetof(uct_mm_iface_config_t, hugetlb_mode), UCS_CONFIG_TYPE_TERNARY},
 
+    {"SEND_ATTEMPTS", "64",
```

Comment:
this is not exactly "send attempts", this is how many endpoints to poll on in every progress iterations.
I think the default should be smaller (e.g 4), and also in runtime it should be min(config.poll_count, iface->num_eps)